kalendar-endlos/src/main/java/com/himanshoe/kalendar/endlos/common/data/KalendarEvent.kt

```diff
@@ -6,10 +6,19 @@ import java.time.LocalDate
  * [KalendarEvent] handles the event marked on any
  * @param[date] with specific
  * @param[eventName] and its
- * @param[eventDescription]
+ * @param[eventDescription] along with it's
+ * @param[eventType]
  */
 data class KalendarEvent(
     val date: LocalDate,
     val eventName: String,
     val eventDescription: String? = null,
+    val eventType: EventType = EventType.EVENT,
 )
+
+sealed class EventType(val type: String) {
+    object GOAL : EventType(type = "GOAL")
```

Comment:
Why do we need "val type: String"??

Reply:
Good catch, let me rewamp this one by creating subclasses instead of objects.

Reply:
Make it like object Goal : EventType()